utils/raster_checker.py

```diff
@@ -0,0 +1,874 @@
+# (c) Nelen & Schuurmans, see LICENSE.rst.
```

Comment:
This file is quite big. Overall it's better to create multiple smaller files with parts of code that are strongly related and/or have a common functionality/dependencies or are logically grouped.

A lot of the functions on the instance don't do anything with data on the instance, so could be exported into external modules within a class or not. (or even as static methods on a class).

This would only make sense if there is a logical grouping of functions into modules/classes, for example the shapefile stuff can probably put in an own module. For the raster checks it might be a bit harder, but maybe there is also some grouping possible.